lightspark-sdk/src/commonJvmAndroidMain/kotlin/com/lightspark/sdk/LightsparkFuturesClient.kt

```diff
@@ -316,6 +316,27 @@ class LightsparkFuturesClient(config: ClientConfig) {
             )
         }
 
+    /**
+     * Gets an estimated amount for the L1 withdrawal fees for the specified node, amount, and strategy.
+     *
+     * @param nodeId The node from where you want to send the payment.
+     * @param amountSats The amount of funds to withdraw in SATOSHI. Use -1 to withdrawal all funds from this node.
+     * @param mode The mode to use for the withdrawal. See `WithdrawalMode` for more information.
+     * @returns An estimate of the fees that will be paid to withdraw funds for the node, amount, and strategy.
+     */
+    suspend fun getWithdrawalFeeEstimate(
```

Comment:
This one shouldn't be a `suspend` function because it's called from Java. It also doesn't need to suspend in this case.